### PR TITLE
enhancement refactor Utility class Constants, Task;

### PR DESCRIPTION
**Pull request by h1alexbel**
Small Style Refactor for Constants Utility class, and Task class.

### CODE REVIEW

File: src/main/software/amazon/event/ruler/Task.java [hunk 1]
```diff
@@ -46,39 +47,39 @@ class Task {
     }
 
     NameState startState() {
-        return machine.getStartState();
+        return this.machine.getStartState();
```
Comment (timbray) [inline src/main/software/amazon/event/ruler/Task.java:50]: Why? (recurs)

File: src/main/software/amazon/event/ruler/Constants.java [hunk 2]
```diff
@@ -4,60 +4,62 @@
...
+final class Constants {
+
+  private Constants() {
+    throw new UnsupportedOperationException();
+  }
+
+  static final String EXACT_MATCH = "exactly";
+  static final String EQUALS_IGNORE_CASE = "equals-ignore-case";
+  static final String PREFIX_MATCH = "prefix";
+  static final String SUFFIX_MATCH = "suffix";
```
Comment (baldawar) [inline src/main/software/amazon/event/ruler/Constants.java:16]: Looks like standard across this pkg is "final static" https://github.com/aws/event-ruler/search?q=static+final, so changing this in just one file would make it inconsistent. In general, I'd favour consistency over ideal practices in just one part of the code.

**Comment by baldawar** [comment]
Appreciate the effort @h1alexbel  but I'm hesitant to take this one in because its not consistent across the pkg. Personally, I'd love for someone to take on https://github.com/aws/event-ruler/issues/43.

File: src/main/software/amazon/event/ruler/Constants.java (hunk 2 above)
Comment (timbray) [reply src/main/software/amazon/event/ruler/Constants.java:16]: > Looks like standard across this pkg is "final static

That would be my fault. Oops.

Comment (h1alexbel) [reply src/main/software/amazon/event/ruler/Constants.java:16]: Hm, if its not expected formatting, can I change all ``` final static ``` to ``` static final```?
@timbray @baldawar

**Comment by h1alexbel** [comment]
@baldawar updated PR with the same pkg ```final + static``` policy as you have. Please take a look.

File: src/main/software/amazon/event/ruler/Task.java (hunk 1 above)
Comment (h1alexbel) [reply src/main/software/amazon/event/ruler/Task.java:50]: using ```this.field``` gives more understanding that this field is related to the class. The same with methods.

Comment (baldawar) [reply src/main/software/amazon/event/ruler/Task.java:50]: I'd recommend to not change this here as nearly all other classes aren't using `this.`

I want the codebase to stay consistent first, and not be deviate unless there's a code reason for one class to be unique on its own.

**Comment by h1alexbel** [comment]
this is the duplication of #61

**Pull request closed without merging**